packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -2417,13 +2417,20 @@ is required to delete the user.
 | id        | `id` of user                                                                          | URL        | String   | TRUE     |         |
 | sendEmail | Sends a deactivation email to the administrator if `true`.  Default value is `false`. | Query      | Boolean  | FALSE    | FALSE   |
 
+> User deletion can also be performed asynchronously. To invoke asynchronous user deletion, pass an HTTP header
+> `Prefer: respond-async` with the request. This header also invokes asynchronous user deactivation when the delete user
```

Comment:
The same _endpoint_ is used to invoke two _different operations_.  Conflating deactivation and delete could sow confusion.  I'd also tweak the wording slightly to emphasize the header is optional.  Proposed tweak:

`... This header is also supported by user activation, which will proceed asynchronously if the delete endpoint is invoked on a user that has not been deactivated`.

Reply:
Note: activation --> deactivation

Reply:
Updated.